lib/python/ensembl/brc4/runnable/load_sequence_data.py

```diff
@@ -791,25 +810,53 @@ def used_cs_ranks(self, agps, cs_order, noagp_default = None):
         return { e:i for i,e in enumerate(sorted(cs_used_set,key=lambda x:-cs_order[x]), start=1) }
 
 
+    def chunk_contigs(self, fasta, cs_ranks, agps, work_dir, chunk_size = 0, chunks_cs_name = "ensembl_internal"):
+        """
+        chunk dna sequence fasta
+          no chunking if chunk_size < 50k
+        """
+        if chunk_size < 50_000:
+            return fasta, cs_ranks, agps
+
+        # split using script
+        en_root = self.param_required("ensembl_root_dir")
+        _splitter = pj(en_root, r"ensembl-genomio/scripts/chunk_fasta.py")
```

Comment:
It'd be a good idea to use paths here, I forgot what pj was supposed to be :)

Reply:
path join

Reply:
Will be done with a separate PR.